src/centralserver/admin-service/src/main/java/org/niis/xroad/centralserver/restapi/openapi/GlobalGroupsApiController.java

```diff
@@ -70,18 +78,23 @@ public ResponseEntity<Void> deleteGlobalGroupMembers(String groupId, Members mem
 
     @Override
     @PreAuthorize("hasAuthority('VIEW_GLOBAL_GROUPS')")
-    public ResponseEntity<Set<GlobalGroup>> findGlobalGroups(String containsMember) {
+    public ResponseEntity<Set<GlobalGroupResource>> findGlobalGroups(String containsMember) {
         return ResponseEntity.ok(globalGroupService.findGlobalGroups(containsMember));
     }
 
     @Override
-    public ResponseEntity<GlobalGroup> getGlobalGroup(String groupId) {
-        throw new NotImplementedException("getGlobalGroup not implemented yet");
+    @PreAuthorize("hasAuthority('VIEW_GROUP_DETAILS')")
+    public ResponseEntity<GlobalGroupResource> getGlobalGroup(String groupId) {
```

Comment:
Mhm, I think I ran off with the primary idea that id could be not just numbers and then later didn't come back when it became apparent it will be only integer not some guid